topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: tutorial_hands_on
+
+title: Peptide Library Data Analysis
+level: Intermediate
+questions:
+  - How to utilize quantitative properties of amino acids and peptide sequence to analyse peptide data?
+objectives:
+  - Calculate descriptors
+  - Quantitative analysis of peptide sequence properties
+time_estimation: '20m'
+contributors:
+   - jaidevjoshi83
+   - blankenberg
+---
+
+# Introduction
+{:.no_toc}
+
+
+Several computational methods have been proven very useful in the initial screening and prediction of peptides for various biological properties. These methods have emerged as effective alternatives to the lengthy and expensive traditional experimental approaches. In this tutorial, we will be discussing how to analyze peptide libraries on the basis of quantitative properties. In this tutorial, we will learn how to use different utilities of PDAUG to calculate various peptide-based features and utilize these features for various informative plots.
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+## Peptide Data
+
+In this step, we will retrieve the inbuild dataset, which contains anti-microbial (AMPs) and transmembrane peptides (TMPs).
+
```

Comment:
Why do we want to look exactly at anti-microbial (AMPs) and transmembrane peptides (TMPs) and compare them? From which organism does the data come from? From which database or where else does the downloaded peptide information come from? These information are also missing in the corresponding Galaxy tool (PDAUG peptide data access).

Reply:
- The objective of this tutorial is to demonstrate how to assess and compare two different peptide libraries based on their features such as charge, hydrophobicity, hydrophobic movement, etc. Several inbuilt data sets have been provided, We have used AMPs and TMPs as an example data set. There is no specific reason two include this particular data set. 
- Detail about the dataset has been included in the text. 
- Tool is also updated with the dataset information.